src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsEngine.java

```diff
@@ -389,15 +382,15 @@ else if (matchingStart.size() == 1) {
      * Creates a UnifiedArgumentCollection with appropriate values filled in from the arguments in this walker
```

Comment:
This comment still talks about UnifiedArgumentCollection

Reply:
fixed